webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -13,13 +13,15 @@
         <hc-user :user="currentUser" :trunc="35" />
         <ds-space />
         <ds-input model="title" class="post-title" placeholder="Title" name="title" autofocus />
+        <small class="smallTag">{{ form.title.length }}/{{ formSchema.title.max }}</small>
         <no-ssr>
           <hc-editor
             :users="users"
             :hashtags="hashtags"
             :value="form.content"
             @input="updateEditorContent"
           />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-09T20:25:08Z" title="Tuesday, July 9th 2019, 10:25:08 pm +02:00">Jul 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -169,6 +169,16 @@ export default {
     },
     updateEditorContent(value) {
       // this.form.content = value
+      this.disabled = true
+      if (
+        value.replace(
+          /<p>|<\/p>|<h3>|<\/h3>|<h4>|<\/h4>|<ul>|<\/ul>|<ol>|<\/ol>|<li>|<\/li>|<blockquote>|<\/blockquote>|<strong>|<\/strong>|<em>|<\/em>|<a.*>|<\/a>/gm,
+          '',
+        ).length > 3
```


Its' quite obscure what you do here. I guess you want to avoid validation errors in the backend? Do we even have a validation for the length of a `post.content`? @ogerly could you double-check?

Anyways, ideally, you would explain what you do here with a test case :+1:

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-07-10T04:49:08Z" title="Wednesday, July 10th 2019, 6:49:08 am +02:00">Jul 10, 2019</time>_

---


no the number of characters in a content is not limited . i would suggest 2000 characters. maybe 4000. you can say badly .. some like to write whole romanes. maybe there should be a BLOG corner for every user for something like that. 
i think 2000 characters is ok. those are 2 A4 pages

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-07-10T07:34:57Z" title="Wednesday, July 10th 2019, 9:34:57 am +02:00">Jul 10, 2019</time>_

---


![FireShot Capture 174 - Human Connection - Human Connection - localhost](https://user-images.githubusercontent.com/1324583/60949675-f73a6f80-a2f5-11e9-95dc-49e72e7b9fd4.png)

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-10T07:54:43Z" title="Wednesday, July 10th 2019, 9:54:43 am +02:00">Jul 10, 2019</time>_

---


But I think we're not talking about maximum but *minimum* characters, right? Or how do you explain what your code does?